changelog/UNRELEASED.md

```diff
@@ -1,12 +1,244 @@
+Contains changes from 4.6.1 and through beta 4.7.1 to 4.7.8.
```

Comment:
Maybe something like `This release consists of changes from the 4.7.x beta releases`

Mentioning 4.6.1 feels a little redundant/obvious